common/tools/dev-tool/src/commands/run/testBrowser.ts

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT license
```

Comment:
Changes in dev-tool are POC right now. Need a way to start the test-server before running the tests, I'm piggybacking on the proxy tool right now. This needs cleanup, also open for suggestions!

Reply:
/cc @willmtemple

Reply:
/cc @witemple-msft

Reply:
I was able to implement tests without the need of TestServer, just mocking!